support/docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM andy5995/netpanzer-build-env:focal
+ARG DEBIAN_FRONTEND=noninteractive
+RUN git clone --depth 1 https://github.com/netpanzer/netpanzer && \
+        cd netpanzer && \
+        meson setup _build && \
+        cd _build && \
+        ninja
+
+FROM ubuntu:focal
```

Comment:
This will be updated to a more recent version. I just used that because I have the focal already available for the appimage build.